python/samples/concepts/agents/azure_ai_agent/azure_ai_agent_declarative_openapi.py

```diff
@@ -24,7 +24,7 @@
 model:
   id: ${AzureAI:ChatModelId}
   connection:
-    connection_string: ${AzureAI:ConnectionString}
+    connection_string: ${AzureAI:Endpoint}
```

Comment:
why is the variable still called connection_string?

Reply:
I missed updating it.

Reply:
Updated